lms/templates/fields/field_dropdown_account.underscore

```diff
@@ -20,6 +20,14 @@
         <span class="u-field-value-readonly"></span>
     <% } else { %>
         <select name="select" id="u-field-select-<%- id %>" aria-describedby="u-field-help-message-<%- id %>">
+            <% if (preOptions) { %>
+                <optgroup label="Common Time Zone">
```

Comment:
This should be plural too, but I'm thought this selection would show all the time zones in the learner's selected country? Already asked above: how is "commonly used" defined? This could be confusing or frustrating to learners if the time zone they want to select isn't deemed "commonly used"?